parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -643,68 +643,73 @@ logs_to_events() {
     fi
 }
 
-should_skip_nvidia_drivers() {
+get_imds_metadata_body() {
```

Comment:
we're specifically calling the `/metadata/instance` endpoint, i'd prefer we rename this to `get_imds_instance_metadata`

Reply:
in the future we might have a need to call other IMDS endpoints other than `/metadata/instance`